package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-skyscanner-with-prettier",
-  "version": "0.4.0",
+  "version": "0.5.0-0",
```

Comment:
I'm not familiar with this semver syntax, is it like an alpha?

Reply:
Yeah I release a pre-release to test out in the backpack repo. After I merge I'll do a proper 0.5.0 release.